tests/regressions/build/CMakeLists.txt

```diff
@@ -0,0 +1,24 @@
+# Copyright (c) 2014-2016 Thomas Heller
+#
+# Distributed under the Boost Software License, Version 1.0. (See accompanying
+# file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+
+add_library(test_server_1950 SHARED server_1950.cpp)
+target_link_libraries(test_server_1950 hpx hpx_init iostreams_component)
```

Comment:
This does not need to be linked with hpx_init. 
Also, what happens in static builds if some target refers to the target `hpx_init`?